ports/rp2/mpconfigport.h

```diff
@@ -281,4 +281,10 @@ extern void lwip_lock_release(void);
 // Bluetooth code only runs in the scheduler, no locking/mutex required.
 #define MICROPY_PY_BLUETOOTH_ENTER uint32_t atomic_state = 0;
 #define MICROPY_PY_BLUETOOTH_EXIT (void)atomic_state;
+
+// Prevent BT+Wifi being active at the same time
+#define hal_check_network_allowed rp2_check_network_allowed
+#define hal_set_network_active rp2_set_network_active
```

Comment:
I think these macros should be called something like `mp_hal_network_check_allowed()` and `mp_hal_network_set_active()`.